packages/sdk/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./create.js";
+export * as waku from "@waku/core";
```

Comment:
Let's export `waitForRemotePeer`, `createDecoder`, `createEncoder` directly from root.
As well as `Decoder`, `Encoder`, `DecodedMessage` and `WakuNode` types